lib/rules/expect-single-argument.js

```diff
@@ -14,7 +14,7 @@ function create (context) {
             message: 'Expect must have a single argument. More than one argument were provided.',
             node
           })
-        } else if (node.arguments.length === 0) {
+        } else if (node.arguments.length === 0 && node.parent.property.name !== 'nothing') {
```

Comment:
@TimZM @tlvince the rule crashes if people just write "expect()" with no arguments and no matcher. This could be fixed by checking if the parent is a MemberExpression or just by optional chaining the property.

Reply:
Indeed you're right. I've created a new PR to fix this issue: #375